frontend/src/lib/data_grid.js

```diff
@@ -92,7 +92,7 @@ var DataGrid = {
             }
 
             var colDef = _.clone(data.cols[cellCol]);
-            colDef['name'] = colDef['display_name'] = formatValue(val, { column: data.cols[pivotCol] });
+            colDef['name'] = colDef['display_name'] = formatValue(val, { column: data.cols[pivotCol] }) || "";
```

Comment:
@tlrobinson can we add a unit test for this?  I think it makes sense to have some coverage on this pivoting function.